docs/modules-in-manual-testcases.md

```diff
@@ -0,0 +1,70 @@
+---
+id: modules-in-manual-testcases
+title: Modules in Manual Testcases
+hide_title: true
+sidebar_label: Link Modules With Manual Testcases
+description: Guide for linking Modules with Manul Testcases in Test Manger.
+keywords:
+  - module creation
+  - import module
+  - test step management 
+  - test case
+  - test steps
+url: https://www.lambdatest.com/support/docs/modules-in-manual-testcases/
+site_name: LambdaTest
+slug: modules-in-manual-testcases/
+---
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "LambdaTest",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "Modules in Manual Testcases",
+          "item": "https://www.lambdatest.com/support/docs/modules-in-manual-testcases/"
+        }]
+      })
+    }}
+></script>
+
+# Link Modules With Manual Test Cases
+***
+Here's how to link and manage modules within your manual test cases:
+
+## 1. Exporting Modules 
```

Comment:
This is not exporting modules, but creation of a module or grouping test steps. UPdate @Manuraj-7